README.md

```diff
@@ -0,0 +1,75 @@
+GFlow
+========================
+
+This is a CLI application that allows datasets and a workflow to be specified in a config file and then have the workflow executed on an instance of Galaxy.
+
+Installation
+------------
+
+Installation right from the source tree (or via pip from PyPI)::
+
+    $ pip install -r requirements.txt
+    $ python setup.py install
+
+Config File
```

Comment:
I would prefer to see config file information in the config file (i.e. descriptive comments and configuration parameter names). I'd keep this section as simple as "rename config.ini.sample to config.ini and modify it according to instructions in the file".